css-cascade-5/Overview.bs

```diff
@@ -1233,8 +1233,8 @@ Declaring Cascade Layers: the ''@layer'' rule</h4>
 	that represents its <dfn>layer name</dfn>.
 	If the [=layer name=] matches that of a [=cascade layer=]
 	already defined within this [=origin=] and [=context=]
-	(and same [=layer=] scope, see [[#nested-layers]]),
-	then its style rules are assigned to that same [=cascade layer=].
+	(and the same [=layer=] scope, see [[#nested-layers]]),
+	then its style rules are assigned to the same [=cascade layer=].
```

Comment:
This "that" is intentional and makes it clear the referent is the preceding cascade layer we mentioned.

```suggestion
	then its style rules are assigned to that same [=cascade layer=].
```